tests/fixtures/config.py

```diff
@@ -106,3 +110,8 @@ class violation_network_access_deny:
         property_group = PropertyGroupData.violation
         name = "network_access"
         granted = False
+
+    class cache:
+        property_group = PropertyGroupData.cache
+        name = "cache_access"
+        granted = False
```

Comment:
This should be granted. Note that `property` does not say what properties exist, but which property groups grant or deny which properties. Properties come to life by being granted or denied.
I'm leaving this because it has no functional relevance in the tests (and these are the legacy fixtures).